### ISSUE TITLE
Please add the Uri* link to Edit pages

### ISSUE THREAD

**Issue by jdaKoko** [Japan-Digital-Archives/Japan-Digital-Archive#861]
My work involves reviewing the archived conditions of items in IA.  Therefore, please add the Uri\* hyperlink (Internet Archive Link) to the edit pages.  See my request in the image.

![uri link request](https://cloud.githubusercontent.com/assets/8831502/4711876/8a3d3f2a-58c2-11e4-9164-4780aebdf8a2.png)

**Comment by corinnecurcie** [comment]
https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/blob/51362f81088abb7f11f6bd0eec5c272aea659632/src/Zeega/AdminBundle/Resources/views/edit_custom.html.twig

**Comment by jcuanm** [comment]
How do we find the "Edit page" if we want to inspect it?

**Comment by jdaKoko** [comment]
log in  http://jdarchive.org/en/login
go to http://jdarchive.org/admin/dashboard
click List "http://jdarchive.org/admin/zeega/data/item/list"
click id numbers at the left side of admin list pages. Then Edit page of the id number opens.
My job requires filtering. For example, "Media Type"  "is equal to" "website" because 
Media Type Wewbsite\* requires Uri\* links in order for me to check wayback pages
Thank you!

**Comment by jcuanm** [comment]
I tried signing into the first link but this page pops up, which doesn't let me do anything
![image](https://cloud.githubusercontent.com/assets/8813674/5021081/4e6f1a20-6a9e-11e4-9c81-8500281c900a.png)

**Comment by jcuanm** [comment]
and then when I try to get into the second link, this page pops up, but I suspect that it has to do with the first link not working:
![image](https://cloud.githubusercontent.com/assets/8813674/5021090/6bbf3e34-6a9e-11e4-8ad1-7d15f43a066b.png)

**Comment by corinnecurcie** [comment]
Try it now. I just added you as an admin, which should give you the
appropriate access.

On Wed, Nov 12, 2014 at 7:02 PM, jcuanm notifications@github.com wrote:

> and then when I try to get into the second link, this page pops up, but I
> suspect that it has to do with the first link not working:
> [image: image]
> https://cloud.githubusercontent.com/assets/8813674/5021090/6bbf3e34-6a9e-11e4-8ad1-7d15f43a066b.png
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/issues/861#issuecomment-62818510
> .

## 

Corinne Curcie
Harvard University
Class of 2015
Computer Science

**Comment by corinnecurcie** [comment]
Okay @jcuanm, I think the dev server should have a functioning admin dashboard now. But you must do the following: 

You have to change your parameters.ini file on the dev server. So once youre in the dev server (don't do this locally, because it won't work: our git system doesn't track the parameters file for security reasons), navigate to MY directory, /var/www/CORINNE, and type sudo vim app/config/parameters.ini - you should be looking at my parameters file. Copy the first 6 database-related lines. Then go into YOUR directory, /var/www/javier, type sudo vim app/config/parameters.ini, which will then bring you to YOUR parameters file, delete the database lines and replace them with the ones I had. 

So basically, your parameters.ini file should look exactly like mine except for the "directory" line. Once you do that, you should be able to access your admin dashboard at http://dev.jdarchive.org/javier/web/admin/dashboard

Then: JUST START MAKING CHANGES. Literally anything you think might work, just start trying stuff and getting errors. It seems daunting at first when you have no idea how the syntax works or what the variables could possibly be, but seriously trying a bunch of stuff helps you figure things out much faster. And get @HakuG to help you!

**Comment by HakuG** [comment]
I've been working on it for a while... Getting the Attribution Uri is a little more annoying than I thought, but It's not bad. Just have to add some javascript code. I have no idea how to get it yet though, because it's in a text, and also the div ids are strangely coded... Will hopefully finish this soon.

**Comment by jcuanm** [comment]
I've been trying to copy the code from the live link URI but that's been
giving me errors. The sonata stuff is getting pretty confusing too... but
yeah we'll get it done soon

**Comment by jdaKoko** [comment]
I just wanted to make sure I would like have the Url\* NOT the Attribution Uri\* on the Edit page.
The Url\* is not a live link.  Each Uri\* has waybackportion http://wayback.archive-it.org/2438/20110301000000 and having waybackportion with the live link leads to the IA archived page.

**Comment by jcuanm** [comment]
We finished it

**Comment by jdaKoko** [comment]
Super, thank you very much y'all.  I  appreciate the effort to add the wayback link. My tasks would be a lot easier!    ＊_(@-@)/_＊
